app/controllers/visits_controller.rb

```diff
@@ -10,9 +10,34 @@ def index
     end
   end
 
+  def show
+    @visit = Visit.find(params[:id])
+    respond_to do |format|
+      if @visit.nil?
```

Comment:
en lugar del if/else en ruby se estila usar el [retorno temprano](http://blog.arkency.com/2014/07/4-ways-to-early-return-from-a-rails-controller/). Como así también los condicionales en una línea [Ej1](http://blog.arkency.com/2014/07/4-ways-to-early-return-from-a-rails-controller/) [Ej2](http://seejohncode.com/2012/07/31/ruby-gotcha-single-line-conditionals/)